js/react_native/android/build.gradle

```
@@ -43,6 +62,22 @@ android {
     sourceCompatibility JavaVersion.VERSION_1_8
     targetCompatibility JavaVersion.VERSION_1_8
   }
+
+  if (getOnnxruntimePackageName() == "onnxruntime-ext") {
+    sourceSets {
+      main {
+        java.srcDirs = ['src/main/java/']
+        java.exclude '**/OnnxruntimeExtensionsDisabled.java'
+      }
+    }
+  } else {
+    sourceSets {
+      main {
+        java.srcDirs = ['src/main/java/']
+        java.exclude '**/OnnxruntimeExtensionsEnabled.java'
+      }
+    }
+  }
```

can we move the condition inside?
```suggestion
  sourceSets {
    main {
      java.srcDirs = ['src/main/java/']
      if (getOnnxruntimePackageName() == "onnxruntime-ext") {
        java.exclude '**/OnnxruntimeExtensionsDisabled.java'
      } else {
        java.exclude '**/OnnxruntimeExtensionsEnabled.java'
      }
    }
  }
```